core/.js/src/main/scala/io/udash/bindings/inputs/Input.scala

```diff
@@ -7,26 +7,27 @@ import scalatags.JsDom.all._
 
 import scala.concurrent.duration.{Duration, DurationInt}
 
-/** Abstraction for HTML input tags.*/
+/** Abstraction for HTML input tags. */
 private[bindings] abstract class Input(inputType: String) {
   /**
-    * @param value Property to bind.
-    * @param debounce Property update timeout after input changes.
-    * @param inputModifiers Additional Modifiers, don't use modifiers on value, onchange and onkeyup attributes.
-    * @return HTML input with bound Property, applied modifiers and nested options.
-    */
-  def apply(value: Property[String], debounce: Duration = 20 millis)(inputModifiers: Modifier*): InputBinding[JSInput] =
+   * @param value               Property to bind.
+   * @param debounce            Property update timeout after input changes.
+   * @param onPropertyUpdated   Callback that's being called when property is updated
+   * @param inputModifiers      Additional Modifiers, don't use modifiers on value, onchange and onkeyup attributes.
```

Comment:
can we improve this comment? add full stop before `don't` and add some explanation